src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp

```diff
@@ -713,20 +655,7 @@ struct GhValenciaDivCleanTemplateBase<
             grmhd::GhValenciaDivClean::BoundaryConditions::BoundaryCondition,
             boundary_conditions>,
         tmpl::pair<gh::gauges::GaugeCondition, gh::gauges::all_gauges>,
-        tmpl::pair<
-            evolution::initial_data::InitialData,
-            tmpl::conditional_t<
-                use_numeric_initial_data,
-                tmpl::flatten<tmpl::list<
-                    grmhd::GhValenciaDivClean::NumericInitialData,
-                    tmpl::conditional_t<
-                        std::is_same_v<SpecInitialData<thermodynamic_dim>,
-                                       NoSuchType>,
-                        tmpl::list<>, SpecInitialData<thermodynamic_dim>>,
-                    tmpl::conditional_t<
-                        std::is_same_v<FukaInitialData, NoSuchType>,
-                        tmpl::list<>, FukaInitialData>>>,
-                analytic_solutions_and_data_to_register>>,
+        tmpl::pair<evolution::initial_data::InitialData, initial_data_list>,
```

Comment:
Can SpEC and FUKA ID remain here?

Reply:
Hm I think this logic should probably be moved to `all_solutions.hpp`.  It looks to me like FUKA wasn't moved correctly and I can fix that

Reply:
I think this logic moved to `all_solutions.hpp`,  I don't think I got the FUKA part yet but the SpEC part should be there.

Reply:
Ok. Please remove the SpEC/FUKA code at the beginning of this file then.